pytest-embedded-nuttx/pytest_embedded_nuttx/__init__.py

```diff
@@ -4,20 +4,23 @@
 
 from pytest_embedded.utils import lazy_load
 
-from .dut import NuttxDut, NuttxEspDut
+from .app import NuttxApp
```

Comment:
`.app` requires `esptool`, which is an optional dependency. so better to keep it in the lazy-load part.

Reply:
Sorry I don't follow what modification is supposed to go here. That line 7 should go away entirely?

Reply:
Thanks!